src/api/app/services/meta_controller_service/request_validator.rb

```diff
@@ -0,0 +1,25 @@
+module MetaControllerService
+  class RequestValidator
+    attr_reader :project, :request_data, :errors
+
+    def initialize(params = {})
+      @project = params[:project]
+      @request_data = params[:request_data]
+      @errors = []
+    end
+
+    def call
```

Comment:
See my other comment. I would prefer to change this method to a private ``validate`` method:

```ruby
class RequestValidator
  def valid?
    validate
    @errors.empty?
  end

  private

  def validate
    # ... the validation code from previous call method
  end
```

This has some advantages:
* keeping the interface as small as possible (one method less in the public interface)
* don't need to remember to call ``call`` before calling ``valid?``
* keeps the same behaviour than ActiveRecord